keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/LogNormal.java

```diff
@@ -8,50 +8,62 @@
 import io.improbable.keanu.distributions.ContinuousDistribution;
 import io.improbable.keanu.distributions.hyperparam.Diffs;
 import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 
 public class LogNormal implements ContinuousDistribution {
```

Comment:
The ContinuousDistribution interface has been removed from the Gaussian distribution. Should it be remove here as well?